tests/e2e/service_invocation/service_invocation_test.go

```diff
@@ -1558,7 +1558,6 @@ func TestNegativeCases(t *testing.T) {
 				// - `rpc error: code = Internal desc = failed to invoke, id: serviceinvocation-callee-0, err: rpc error: code = Internal desc = error invoking app channel: Post \"http://127.0.0.1:3000/timeouterror\": context deadline exceeded``
 				// - `rpc error: code = DeadlineExceeded desc = context deadline exceeded`
 				assert.Contains(t, testResults.RawError, "rpc error:")
-				assert.Contains(t, testResults.RawError, "RST_STREAM with error code: CANCEL")
```

Comment:
why this change and the one in mtls.go?

Reply:
I believe this string check is no longer correct because of a lib dep upgrade, I have updated the string check above to check for status code check via string match.